src/commands/e2e/WdioConfigTemplate.ts

```diff
@@ -273,6 +274,26 @@ exports.config = {
             blockOutStatusBar: true,
             blockOutToolBar: true,
         }],`;
+    }
+
+    protected getSeleniumStandalone(): string {
+        // tslint:disable-next-line:no-any
+        const seleniumConfig: { [k: string]: any } = {};
```

Comment:
👍  I personally also prefer generation via JSON object instead of string templates